modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaMultiVMPoolUtilCheck.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeJavaMultiVMPoolUtilCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		if (!fileName.endsWith(".java")) {
+			return content;
+		}
+
+		if (content.contains(_UTIL_IMPORT)) {
```

Comment:
@nicolas-sss, I liked your idea to add this import to a constant, good job! :clap: 

Even though, this name is too generic, hence isn't very clear which util class you're referencing, does it make sense to you?

In addition, you added the `MultiVMPoolUtil` to a constant, but didn't do the same to the new class, the `MultiVMPool` which can be seen as a lack of consistency from Brian's perspective.

Can you please use here the full name of the imports or put both of them in constants with meaningful names?

```
		if (content.contains(_MULTI_VM_POOL_UTIL_IMPORT)) {
			content = StringUtil.replace(
				content, _MULTI_VM_POOL_UTIL_IMPORT,
				_MULTI_VM_POOL_IMPORT);
			content = _replaceGetPortalCache(content, fileName);
		}
```
OR
```

		if (content.contains("import com.liferay.portal.kernel.cache.MultiVMPoolUtil;")) {
			content = StringUtil.replace(
				content, "import com.liferay.portal.kernel.cache.MultiVMPoolUtil;",
				"import com.liferay.portal.kernel.cache.MultiVMPool;");
			content = _replaceGetPortalCache(content, fileName);
		}
```

Reply:
We agree that we only create constants when we have at least 2 uses for them. This is only the case for MultiVMPoolUtil import.